src/ImageGallery.js

```diff
@@ -427,9 +427,18 @@ class ImageGallery extends React.Component {
 
     // don't show some slides while transitioning to avoid background transitions
     const isVisible = this.isSlideVisible(index);
+    let hidden;
+    if (isCanvas === true) {
+      hidden =
+          'isCanvas' in items[this.getCurrentIndex()] && items[this.getCurrentIndex()].isCanvas
```

Comment:
Is this PR related to #691 ?

If so, this should not be affecting the issue you're having. The display logic here hides slides that should not show during the transition.

Reply:
yes, It's related to #691. when there is 3D graphics, It gets hidden due to transition and the height of the canvas element sets to 0. once the canvas element height is 0 it disappears from the page and when we again click on the 360 icon It shows blank.  I hope this will help you!